src/components/WorkspaceAdd/WorkspaceAddModal.test.tsx

```diff
@@ -286,20 +251,22 @@ describe('Add Workspace modal', () => {
         },
       },
     });
-    await waitFor(() => expect(screen.queryByRole('dialog')).not.toBeInTheDocument());
+    await waitFor(() => expect(onCloseMock).toBeCalledTimes(1));
+    // Uncomment when navigation to details page has been implemented in WorkspaceAddModal
```

Comment:
We're prioritizing the workspace details PR #25 first - so we will have to follow up or rebase this PR later

Reply:
I removed the comment.